components/admin_console/permission_schemes_settings/permissions_tree.jsx

```diff
@@ -68,6 +68,7 @@ const GROUPS = [
             'manage_webhooks',
             'manage_oauth',
             'manage_slash_commands',
+            'manage_emojis',
```

Comment:
should `manage_others_emojis` appear somewhere around here too?

Reply:
nope. This is what is shown in the admin interface. The old behavior was to allow manage your own emojis. So this is what is exposed (for now) in the current interface (It mimics the behavior of other things like integrations).

Reply:
:+1: